be/src/runtime/load_stream.cpp

```diff
@@ -178,6 +179,19 @@ Status TabletStream::add_segment(const PStreamHeader& header, butil::IOBuf* data
     DCHECK(header.has_segment_statistics());
     SegmentStatistics stat(header.segment_statistics());
     TabletSchemaSPtr flush_schema;
+    DataDir* data_dir = nullptr;
+
+    TabletSharedPtr tablet = StorageEngine::instance()->tablet_manager()->get_tablet(header.tablet_id());
+    if (tablet != nullptr) {
+        data_dir = tablet->data_dir();				 
+    }
+
+    if (data_dir != nullptr &&
+        data_dir->reach_capacity_limit(stat.data_size + stat.index_size)) {
+        LOG(INFO) << "segment size is " << (stat.data_size + stat.index_size);
```

Comment:
maybe debug log is better.
After disk usage is high, lots of these logs will take large disk space.

Reply:
already change to DEBUG